swatch-core/src/main/java/org/candlepin/subscriptions/clowder/ClowderJsonEnvironmentPostProcessor.java

```diff
@@ -20,97 +20,37 @@
 import org.springframework.core.io.Resource;
 import org.springframework.core.io.ResourceLoader;
-import org.springframework.util.ClassUtils;
-import org.springframework.util.CollectionUtils;
 import org.springframework.util.StringUtils;
-import org.springframework.web.context.support.StandardServletEnvironment;
 
 /**
- * An {@link org.springframework.boot.env.EnvironmentPostProcessor} that takes Clowder JSON and
- * converts it into a form easily interoperable with the rest of the Spring configuration. The
- * Clowder JSON is flattened into properties where it can be referenced elsewhere and each property
- * is prefixed with the word "clowder".
- *
- * <p>For example:
- *
- * <pre><code>
- * { "kafka": {
- *   "brokers": [{
- *     "hostname": "localhost",
- *    }]
- * }}
- * </code></pre>
- *
- * becomes <code>clowder.kafka.brokers[0].hostname</code>.
- *
- * <p>These properties can be referenced elsewhere in application configuration using the "${}"
- * syntax and can be assigned to intermediate properties such as environment variables. This allows
- * usages like
- *
- * <pre><code>
- *   rhsm.kafka.host: ${KAKFA_HOST}
- *   KAFKA_HOST: ${clowder.kafka.brokers[0].hostname}
- * </code></pre>
- *
- * This use of an intermediate environment variable allows for succinct overriding during local
- * application development when the Clowder JSON is not available.
- *
- * <p>Properties that exist in both the Clowder JSON are at a high precedence in the property
- * resolution order. For example, if "clowder.kafka.brokers[0].hostname" is defined in
- * application.yaml, as a Java system property, and in the Clowder JSON, the value from the Clowder
- * JSON will win. Only command line arguments and property values defined in tests have higher
- * precedence.
- *
- * <p>This class is inspired by (and derived from) {@link
- * org.springframework.boot.cloud.CloudFoundryVcapEnvironmentPostProcessor} and {@link
- * org.springframework.boot.env.SpringApplicationJsonEnvironmentPostProcessor}
+ * An {@link org.springframework.boot.env.EnvironmentPostProcessor} that inserts a {@link
+ * ClowderJsonPathPropertySource} into the list of property sources.
  */
 public class ClowderJsonEnvironmentPostProcessor implements EnvironmentPostProcessor, Ordered {
   public static final String JSON_RESOURCE_LOCATION =
       "rhsm-subscriptions.clowder.json-resource-location";
 
-  private static final String SERVLET_ENVIRONMENT_CLASS =
-      "org.springframework.web.context.support.StandardServletEnvironment";
-
-  private static final Set<String> SERVLET_ENVIRONMENT_PROPERTY_SOURCES =
-      new LinkedHashSet<>(
-          Arrays.asList(
-              StandardServletEnvironment.JNDI_PROPERTY_SOURCE_NAME,
-              StandardServletEnvironment.SERVLET_CONTEXT_PROPERTY_SOURCE_NAME,
-              StandardServletEnvironment.SERVLET_CONFIG_PROPERTY_SOURCE_NAME));
+  public static final String CLOWDER_STRICT_LOADING = "rhsm-subscriptions.clowder.strictLoading";
```

Comment:
Nitpick: Spring best practice is lower-case kebab format according to [the tip in Spring docs](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.typesafe-configuration-properties.relaxed-binding) (note that #624 should be adjusted if you change it).
```suggestion
  public static final String CLOWDER_STRICT_LOADING = "rhsm-subscriptions.clowder.strict-loading";
```